Source/Private/ASTextNode2.mm

```diff
@@ -11,6 +11,7 @@
 //
 
 #import <AsyncDisplayKit/ASTextNode2.h>
+#import <AsyncDisplayKit/ASTextNode.h>  // Definition of ASTextNodeDelegate
```

Comment:
In the future everything will be merged. However, in a separate diff, we could clean this up to add an ASTextNodeDefinitions.h or similar to declare both the highlight enum and the delegate protocol.

I'm aiming to keep this particular diff quick, but it does raise the question of what our migration path should be to ASTextNode2.